smtp/main.go

```diff
@@ -46,8 +46,8 @@ type smtpNotifier struct {
 }
 
 type mailConfig struct {
-	server, port, sender, password string
-	recipients                     []string
+	server, port, sender, from, password string
+	recipients                           []string
 }
```

Comment:
Bad merge here? Are you working on a new branch?

Reply:
Resolved!